libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/Queue/CSFNetwork+Salesforce.m

```diff
@@ -39,17 +39,18 @@ - (void)setDefaultConnectCommunityId:(NSString *)defaultConnectCommunityId {
 - (void)setupSalesforceObserver {
     [[NSNotificationCenter defaultCenter] addObserver:self
                                              selector:@selector(userAccountManagerDidChangeCurrentUser:)
```

Comment:
Can we change the name of the selector as well?  It doesn't reflect the nature of the notification anymore.